app/src/main/java/com/example/keniphhomemaintenance/UtilitiesScreen.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.keniphhomemaintenance
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+@Composable
+fun UtilitiesScreen(name: String?, address: String?) {
+    Box(contentAlignment = Alignment.Center, modifier = Modifier.fillMaxSize()) {
+        Text(text = "Name of house is $name, and its address is $address")
```

Comment:
Text in strings.xml

Reply:
added, but this will get deleted when booni's branch merges